packages/camera/camera_platform_interface/lib/src/types/video_stabilization_mode.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The possible video stabilization modes that can be capturing video.
+enum VideoStabilizationMode {
+  /// Video stabilization is disabled.
+  off,
+
+  /// Basic video stabilization is enabled.
+  ///
+  /// Maps to CONTROL_VIDEO_STABILIZATION_MODE_ON on Android
+  /// and throws CameraException on iOS.
+  on,
+
+  /// Standard video stabilization is enabled.
+  ///
+  /// Maps to CONTROL_VIDEO_STABILIZATION_MODE_PREVIEW_STABILIZATION on Android
+  /// (camera_android_camerax) and to AVCaptureVideoStabilizationModeStandard
+  /// on iOS.
+  standard,
+
+  /// Cinematic video stabilization is enabled.
+  ///
+  /// Maps to CONTROL_VIDEO_STABILIZATION_MODE_PREVIEW_STABILIZATION on Android
+  /// (camera_android_camerax) and to AVCaptureVideoStabilizationModeCinematic
+  /// on iOS.
+  cinematic,
+
+  /// Extended cinematic video stabilization is enabled.
```

Comment:
This seems extremely closely tied to the specific iOS implementation. It would be much better to give a more generic description of what this is (which based on the iOS docs seems to be a better but slower/more expensive stabilization algorithm), so that it is clearer what it would mean for another platform to support this.